google/cloud/internal/grpc_opentelemetry.h

```diff
@@ -58,6 +59,24 @@ opentelemetry::nostd::shared_ptr<opentelemetry::trace::Span> MakeSpanGrpc(
  */
 void InjectTraceContext(grpc::ClientContext& context, Options const& options);
 
+/**
+ * Extracts information from the `grpc::ClientContext`, and adds it to a span.
+ *
+ * The span is ended. The original value is returned, for the sake of
+ * composition.
+ *
+ * Note that this function should be called after the server has filled in the
```

Comment:
Offline we discussed something like "after the RPC has finished" instead of "after the server has filled ...", seeing as the server isn't doing anything on the client side.